_assets/sass/custom/_buttons.scss

```diff
@@ -35,10 +35,11 @@
 .crt-page--downloadpdf-link {
   align-items: baseline;
   display: inline-flex;
-}
+  gap: 8px;
```

Comment:
I didn't know this was a thing! Super useful!

Reply:
😄 [I definitely recommend this guide](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) - it's definitely worth skimming the pictures, I go back to it pretty much every time I use flex.

Reply:
these pics are so useful